packages/swapkit/chainflip/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "author": "swapkit",
+  "dependencies": {
+    "@chainflip/sdk": "1.1.5",
+    "@polkadot/api": "10.11.2",
+    "@polkadot/keyring": "^12.6.2",
+    "@polkadot/util": "^12.6.2"
```

Comment:
remove `^` from deps and update to newest - we don't want unexpected updates of versions